src/redux/actions/projects/createProject.js

```diff
@@ -0,0 +1,28 @@
+import moment from 'moment';
+import { v4 as uuidv4 } from 'uuid';
+
+import {
+  PROJECTS_CREATE,
+} from '../../actionTypes/projects';
+import { projectTemplate } from '../../reducers/projects/initialState';
+
+const createProject = (
+  projectName,
+) => async (dispatch) => {
+  const createdAt = moment().toISOString();
+
+  const newProject = {
+    ...projectTemplate,
+    name: projectName,
+    uuid: uuidv4(),
```

Comment:
what is the relationship between this `uuid` in here and `experimentId`? Where and what for is `uuid` used? Where do we record the experiment id?

Reply:
Oh yea, I'll add an array to store the experiments related to the project. Each experiment will also have a `projectUuid` to point back to the parent project,

Reply:
updated in c5355d9